go.mod

```diff
@@ -41,4 +41,6 @@ replace (
 	k8s.io/kube-openapi => k8s.io/kube-openapi v0.0.0-20200204173128-addea2498afe // avoids case-insensitive import collision: "github.com/googleapis/gnostic/openapiv2" and "github.com/googleapis/gnostic/OpenAPIv2"
 )
 
+replace github.com/codeready-toolchain/api => github.com/xcoulon/api v0.0.0-20200630081021-4d177ed6d634
```

Comment:
A reminder :-)

Reply:
removed in https://github.com/codeready-toolchain/toolchain-common/commit/fdaf9a060f38f5785b1af3518f0c9a16e96757f5